tests/unary_classification_with_text_gen.rs

```diff
@@ -0,0 +1,1130 @@
+    detector_mocks.mock(|when, then| {
+        when.post()
+            .path(TEXT_CONTENTS_DETECTOR_ENDPOINT)
+            .header("x-model-name", DETECTOR_NAME_ANGLE_BRACKETS_SENTENCE)
+            .json(ContentAnalysisRequest {
+                contents: vec![
+                    expected_response.generated_text.clone()
+                ],
+                detector_params: DetectorParams::new(),
+            });
+        then.json(vec![Vec::<ContentAnalysisResponse>::new(),]);
+    });
+  
+    // Add chunker tokenization for input mock
+    chunker_mocks.mock( |when, then| {
+        when.path(CHUNKER_UNARY_ENDPOINT)
+            .header(CHUNKER_MODEL_ID_HEADER_NAME, CHUNKER_NAME_SENTENCE)
+            .pb(ChunkerTokenizationTaskRequest {
+                text: text_mock_input.clone(),
+            });
+        then.pb(TokenizationResults {
+            results: vec![
+                Token {
+                    start: 0,
+                    end: 22,
+                    text: text_mock_input.clone(),
+                },
+            ],
+            token_count: 0,
+        });
+    });
+
+    // Add chunker tokenization for generated text
+    chunker_mocks.mock( |when, then| {
+        when.path(CHUNKER_UNARY_ENDPOINT)
+            .header(CHUNKER_MODEL_ID_HEADER_NAME, CHUNKER_NAME_SENTENCE)
+            .pb(ChunkerTokenizationTaskRequest {
+                text: expected_response.generated_text.clone(),
+            });
+        then.pb(TokenizationResults {
+            results: vec![
+                Token {
+                    start: 0,
+                    end: 11,
+                    text: expected_response.generated_text.clone(),
+                },
+            ],
+            token_count: 0,
+        });
+    });
+
+    // Configure mock servers
+    let mock_detector_server = MockServer::new(DETECTOR_NAME_ANGLE_BRACKETS_SENTENCE).with_mocks(detector_mocks);
+    let mock_generation_server = MockServer::new("generation_server").grpc().with_mocks(generation_mocks);
+    let mock_chunker_server = MockServer::new(CHUNKER_NAME_SENTENCE.into()).grpc().with_mocks(chunker_mocks);
+
+    // Run test orchestrator server
+    let orchestrator_server = TestOrchestratorServer::builder()
+        .config_path(ORCHESTRATOR_CONFIG_FILE_PATH)
+        .chunker_servers([&mock_chunker_server])
+        .detector_servers([&mock_detector_server])
+        .generation_server(&mock_generation_server)
+        .build()
+        .await?;
+
+    // Orchestrator request with no input/output detectors for no detectors scenario
+    let response = orchestrator_server
+        .post(ORCHESTRATOR_UNARY_ENDPOINT)
+        .json(&GuardrailsHttpRequest {
+            model_id: MODEL_ID.into(),
+            inputs: text_mock_input.clone(),
+            guardrail_config: None,
+            text_gen_parameters: None,
+        })
+        .send()
+        .await?;
+
+    // Assertions on no detectors scenario
+    assert!(response.status() == StatusCode::OK);
```

Comment:
Whenever possible, replace `assert!(something == something_else` with `assert_eq!(something, something_else)` for better error logging.
```suggestion
    assert_eq!(response.status(), StatusCode::OK);
```

assert!():
![image](https://github.com/user-attachments/assets/e3c21c84-fe40-4a19-87ce-51ff7b039388)


assert_eq!():
![image](https://github.com/user-attachments/assets/28758e8d-c00f-47f3-b26d-ea35340d5f69)

Reply:
Thanks, I'll replace it throughout assertions.